src/pydiscourse/client.py

```diff
@@ -604,6 +605,87 @@ def delete_topic(self, topic_id, **kwargs):
 
         """
         return self._delete(f"/t/{topic_id}", **kwargs)
+    
+    def _topic_attachments_urls(self, slug, topic_id, **kwargs):
+        """
+
```

Comment:
Please add example comment, eg "Extract URLs of attachments from every post in topic"